cookbook/bundles/dependencies.rst

```diff
@@ -0,0 +1,51 @@
+.. index::
+   single: Bundle; Dependencies
+
+How to Use Bundle Dependencies to load other Bundles
+====================================================
+
+When working on your own bundle(s), you'll sometimes have the need to reuse
+other bundles, either by just requiring, overriding or inherit from them.
+
+While Composer takes care about letting you make sure these dependencies are
+loaded, you'll also need to enable these bundles in the Kernel.
+
+If your bundle is meant for reuse, then this will typically complicate your
+installation documentation, and if your dependencies change over time they
+will likewise complicate your upgrade documentation, and make upgrading
+more tedious for your users.
+
+Starting with Symfony 2.7 you can avoid this by specifying your dependencies
+by implementing "BundleDependenciesInterface" interface::
+
+    // src/Acme/UserBundle/AcmeCacheBundle.php
+    namespace Acme\CacheBundle;
```

Comment:
Ok, note that this is the case in [inheritance.rst](https://github.com/andrerom/symfony-docs/blob/patch-1/cookbook/bundles/inheritance.rst)